main/controllers/user/city_coordinates.py

```diff
@@ -0,0 +1,28 @@
+from main import db
+from main.controllers.user import user_bp, API_CATEGORY
+from flask_apispec import marshal_with, doc
+from main.controllers.common.common import convert_query_to_response
+from main.models.common.error import ResponseError
+from main.models.city import t_city
+from main.models.schema.user import ResponseCityCoordinatesSchema
+from sqlalchemy import func
+
+@user_bp.route("/city_coordinates", methods=["GET"])
```

Comment:
REST API conventions dictate underscores should not be used. let's change to dashes :)